src/fs/reader.ts

```diff
@@ -14,34 +14,21 @@ export interface IReadOptions {
 
 export async function readFile(name: string | number, opts: IReadOptions): Promise<string> {
   if (typeof name === 'number') {
-    let result = '';
-
-    const stream = fs.createReadStream('', { fd: name });
-    stream.setEncoding(opts.encoding);
-
-    stream.on('readable', () => {
-      let chunk: string | null;
-
-      while ((chunk = stream.read()) !== null) {
-        result += chunk;
-      }
-    });
+    // Piped through STDIN
+    return readFrom(name, opts.encoding);
```

Comment:
@P0lip This doesn't fail on windows, whereas the `createReadStream` cringes with "EOF found"